runtime/util/rcdump.c

```diff
@@ -933,9 +955,14 @@ I_32 j9bcutil_dumpRomMethod( J9ROMMethod *romMethod, J9ROMClass *romClass, J9Por
 
 	j9tty_printf( PORTLIB, "  Access Flags (%X): ", romMethod->modifiers);
 	printModifiers(PORTLIB, (U_32)romMethod->modifiers, INCLUDE_INTERNAL_MODIFIERS, MODIFIERSOURCE_METHOD);
+	if (J9_ARE_ALL_BITS_SET(romClass->optionalFlags, J9_ROMCLASS_OPTINFO_IMPLICITCREATION_ATTRIBUTE)) {
+		if ((6 == J9UTF8_LENGTH(J9ROMMETHOD_NAME(romMethod))) && strcmp((char *)J9UTF8_DATA(J9ROMMETHOD_NAME(romMethod)), "<vnew>")) {
```

Comment:
You can use one of the existing macros to do the comparison: https://github.com/eclipse-openj9/openj9/blob/d1eab2b63578a71b7d951c51dc6f25f032a28c04/runtime/oti/j9.h#L177-L185